tests/software/bedtools/intersect/main.nf

```diff
@@ -9,6 +9,19 @@ workflow test_bedtools_intersect {
               file(params.test_data['sarscov2']['genome']['test_bed'], checkIfExists: true),
               file(params.test_data['sarscov2']['genome']['test2_bed'], checkIfExists: true)
```

Comment:
`test2_bed` should be updated on test-datasets now and will have the contents you generated.